sormas-backend/src/main/java/de/symeda/sormas/backend/visit/VisitFacadeEjb.java

```diff
@@ -462,9 +465,25 @@ public List<VisitExportDto> getVisitsExportList(
 			if (resultList.size() > 0) {
 
 				Pseudonymizer pseudonymizer = Pseudonymizer.getDefault(userService::hasRight);
+				Map<Long, UserReference> visitUsers =
+					userService
+						.getUserReferencesByIds(
+							resultList.stream()
+								.map(c -> Arrays.asList(c.getVisitUserId()))
+								.flatMap(Collection::stream)
+								.filter(Objects::nonNull)
+								.collect(Collectors.toSet()))
+						.stream()
+						.collect(Collectors.toMap(UserReference::getId, Function.identity()));
 				for (VisitExportDto exportDto : resultList) {
 					boolean inJurisdiction = exportDto.getInJurisdiction();
 
+					UserReference user = visitUsers.get(exportDto.getVisitUserId());
+
+					exportDto.setVisitUserName(user.getName());
+					exportDto.setVisitUserRoles(
+						user.getUserRoles().stream().map(userRole -> UserRoleFacadeEjb.toReferenceDto(userRole)).collect(Collectors.toSet()));
```

Comment:
`userRole -> UserRoleFacadeEjb.toReferenceDto(userRole) ` can be simplified to `UserRoleFacadeEjb::toReferenceDto `

Reply:
true